resources/assets/components/Users/UserInformation.js

```diff
@@ -43,53 +53,36 @@ const UserName = ({ user, link }) => {
   return <span>{displayName}</span>;
 };
 
-const UserInformation = props => (
+export default ({ user, linkSignup, children }) => (
```

Comment:
Is this `default` preferred over our usual approach of exporting the component names explicitly?

Reply:
I don't have much of a preference, but I think you're right that we tend to name and export! I'll keep that convention going here. 👍

Reply:
Changed in 0b63d480.